internal/fs/inode/file.go

```diff
@@ -46,10 +46,11 @@ type FileInode struct {
 	// Constant data
 	/////////////////////////
 
-	id           fuseops.InodeID
-	name         Name
-	attrs        fuseops.InodeAttributes
-	contentCache *contentcache.ContentCache
+	id             fuseops.InodeID
+	name           Name
+	attrs          fuseops.InodeAttributes
+	contentCache   *contentcache.ContentCache
+	localFileCache bool
```

Comment:
Leave a TODO here to remove this bool flag by refactoring the content cache using two different implementations.